.github/workflows/cd.yml

```diff
@@ -38,9 +38,12 @@ jobs:
       - name: Run linter 👀
         run: npm run lint
 
-      - name: Run tests 🧪
+      - name: Run Unit Tests 🧪
         run: npm run test
 
+      - name: Run E2E Tests 🧪
```

Comment:
We may need to change the order of this later, so we can run the tests after generating the static site. Let's do it when we have some real tests 👍